### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by theodesp**
Typos and some test case strings

### CODE REVIEW

File: stubs_test.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ func (ni *NullInt) Scan(value interface{}) error {
 		ni.Integer, ni.Valid = 0, false
 	case int64:
 		const maxUint = ^uint(0)
-		const minUint = 0
```
Comment (theodesp) [inline stubs_test.go]: Unused

**Comment by l3pp4rd** [comment]
Hi, I think you made a type somewhere and it is not longer parseable source code, check the travis report

File: rows.go [hunk 2]
```diff
@@ -9,9 +9,9 @@ import (
 )
 
 // CSVColumnParser is a function which converts trimmed csv
-// column string to a []byte representation. currently
+// column string to a []byte representation. Currently
 // transforms NULL to nil
-var CSVColumnParser = func(s string) []byte {
+var CSVColumnParser = func(s string) []bytezzx {
```
Comment (theodesp) [inline rows.go]: oops

**Comment by l3pp4rd** [comment]
thanks ;)

**Pull request merged**